books/american_government/bake

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require 'bundler/inline'
+
+gemfile do
+  gem 'openstax_kitchen', ENV['USE_LOCAL_KITCHEN'] ? { path: '/code/kitchen' } : '7.0.0'
+  gem 'slop', '4.8.2'
+  gem 'byebug'
+end
+
+recipe = Kitchen::BookRecipe.new(book_short_name: :american_gov) do |doc|
+  include Kitchen::Directions
+
+  book = doc.book
+  metadata = book.metadata
+
+  # Some stuff just goes away
+  book.search('cnx-pi').trash
+
+  BakePreface.v1(book: book, title_element: 'div')
+  BakeUnitTitle.v1(book: book)
+  BakeChapterTitle.v1(book: book)
+  BakeChapterIntroductions.v1(book: book)
+  BakeUnclassifiedNotes.v1(book: book)
+
+  book.chapters.each do |chapter|
+    BakeChapterGlossary.v1(chapter: chapter, metadata_source: metadata)
+    BakeChapterSummary.v1(chapter: chapter, metadata_source: metadata)
+    BakeNonIntroductionPages.v1(chapter: chapter)
+
+    chapter.search('.review-questions, .critical-thinking').exercises.each do |exercise|
```

Comment:
Can these have the element name?  Element name searches are faster.  E.g. if they are always `div`...

```ruby
chapter.search(%w[review-questions critical-thinking].prefix('div.'))
```

Karina added the `prefix` method

... actually should have a line that is 

```ruby
exercise_section_classes = %w[review-questions critical-thinking]
```

and use that

```ruby
chapter.search(exercise_section_classes.prefix('div.'))
```

to help with below